docs/millennium-os/manual/chapters/20_installation.md

```diff
@@ -8,19 +8,19 @@ MillenniumOS for RRF is distributed as a **single Zip file** that contains every
 ## RRF Installation
 
 !!! note
-    If you installed one of our [RRF Configurations](https://github.com/MillenniumMachines/RRF-Configs/releases) that contains MillenniumOS, then you may not need to follow everything in this guide. If you start RRF and a MillenniumOS configuration wizard appears, then MillenniumOS is already installed and just needs to be [configured](#configuration)!.
+    If you installed one of our [RRF Configurations](https://github.com/MillenniumMachines/RRF-Configs/releases) that contains MillenniumOS, then you may not need to follow everything in this guide. If you start RRF and a MillenniumOS configuration wizard appears, then MillenniumOS is already installed. It just needs to be [configured](#configuration)!
 
 ### Download MillenniumOS Release
 
-MillenniumOS releases can be found on the [MillenniumOS Releases](https://github.com/MillenniumMachines/MillenniumOS/releases) page. Select the `mos-release-vX.X.X...zip` file to download the latest release.
+MillenniumOS releases are on the [MillenniumOS Releases](https://github.com/MillenniumMachines/MillenniumOS/releases) page. Select the `mos-release-vX.X.X...zip` file to download the latest release.
```

Comment:
"are found"?

Reply:
Yeah, I agree with you, even though Grammarly doesn't. 
![xKT17xo 1](https://github.com/user-attachments/assets/bb3bdc7a-2cb4-4f65-aa4f-e737cc5aeab3)

Changing it to your way.